client/api/eventsApi.js

```diff
@@ -25,12 +25,20 @@ export function addNewEvent (id, event) {
 }
 
 /**
- * @typedef {object} Event
- * @property {number} id
+ * Event object details
+ * @typedef {object<string, any>} Event
```

Comment:
```suggestion
 * @typedef {object.<string, any>} Event
```
💭 JSDoc syntax is a bit weird, but c'est la vie 🤷🏻‍♂️